### ISSUE TITLE
Cannot make this work with Expo

### ISSUE THREAD

**Issue by dolsem** [jkzing/vscode-redux-devtools#11]
I am debugging with Expo and a real iOS device. If I understand the docs correctly, I need to run remotedev-server, which I did by creating the following file and running with Node:
`
const remotedev = require('remotedev-server');
remotedev({
  hostname: 'MY_LAN_IP',
  port: 19080,
});
`

I also changed my store enhancers like so:
`
  composeEnhancers =
    require('redux-devtools-extension').composeWithDevTools({
      realtime: true,
      name: 'Test App',
      hostname: 'MY_LAN_IP',
      port: 19080,
      serialize: {
        options: {
          symbol: true,
        },
      },
    });
`
I used to debug with  React Native Debugger, and I only had 'serialize' property defined. I did not have to run a remotedev-server instance. Everything was working fine. 

I run packager, then open the app on my phone and go to http://MY_LAN_IP:19080 in Chrome, it opens up the monitor, but everything is empty and the state is undefined. Needless to say, pressing Connect in the VS extension after setting hostname and port also does nothing.

Is this a bug in remotedev-server or something else, or did I not set it up properly?

**Comment by jkzing** [comment]
Apologize for the delay of response, I just have been really busy on my day job...

Actually I haven't used this with React Native yet. 😅 
I suppose the RN debugger would start a `remotedev-server` instance, all you have to do is to connect the devtool with it.

Seems you can find how to configure the host and port here: [https://github.com/jhen0409/react-native-debugger/blob/master/docs/getting-started.md](https://github.com/jhen0409/react-native-debugger/blob/master/docs/getting-started.md) (I guess...). Hope it can help you.

**Comment by dolsem** [comment]
@jkzing yes, it was my understanding that RND runs its own instance. I'm trying to get rid of RND though and use VS instead. So in order to make the plugin work, I need to setup the server correctly. As far as I understand, I don't have to change redux devtools extension setup, just run remotedev-server. However, running my code above doesn't let me connect to the server. Can you please tell me if something's wrong with the way I run the server in my code, or if there are any extra steps? I'm assuming RN should not be different from pure React in this context?

**Comment by jkzing** [comment]
@dolsem Seems `redux-devtools-extension` just support local inspection, could you try `remote-redux-devtools` instead?
